src/ansys/fluent/core/launcher/launcher_arguments.py

```diff
@@ -0,0 +1,346 @@
+from enum import Enum
+import os
+from pathlib import Path
+import platform
+import subprocess
+import tempfile
+from typing import Any, Dict, Optional, Union
+
+from ansys.fluent.core.exceptions import DisallowedValuesError
+from ansys.fluent.core.fluent_connection import FluentConnection, PortNotProvided
+from ansys.fluent.core.launcher.custom_exceptions import (
+    DockerContainerLaunchNotSupported,
+    InvalidPassword,
+    IpPortNotProvided,
+    UnexpectedKeywordArgument,
+)
+from ansys.fluent.core.launcher.launcher_utils import check_docker_support, logger
+from ansys.fluent.core.session import _parse_server_info_file
+from ansys.fluent.core.session_meshing import Meshing
+from ansys.fluent.core.session_pure_meshing import PureMeshing
+from ansys.fluent.core.session_solver import Solver
+from ansys.fluent.core.session_solver_icing import SolverIcing
+from ansys.fluent.core.utils.fluent_version import FluentVersion
+from ansys.fluent.core.utils.networking import find_remoting_ip
+import ansys.platform.instancemanagement as pypim
+
+
+class LaunchMode(Enum):
+    """An enumeration over supported Fluent launch modes."""
```

Comment:
Our standard is to start class descriptions with a verb ending is "s" or "es". I haven't commented on this earlier, but this is the standard that most PyAnsys libraries are following.